src/OpenTelemetry/Internal/OpenTelemetrySdkEventSource.cs

```diff
@@ -309,6 +328,18 @@ public void MissingPermissionsToReadEnvironmentVariable(string exception)
             this.WriteEvent(30, exception);
         }
 
+        [Event(31, Message = "'{0}<{1}>' exporting to '{2}' dropped '0' items.", Level = EventLevel.Informational)]
```

Comment:
see this : https://github.com/open-telemetry/opentelemetry-dotnet/pull/2331/files#r711278336
if you do that, you won need to do this stitching here.